CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## 1.17.1-dev
+## 1.17.1
 * Bug - Fixed a bug that was causing a runtime panic by accessing negative index in the health check log slice. [#1239](https://github.com/aws/amazon-ecs-agent/pull/1239)
```

Comment:
Can you break this line to 80 characters and also get rid of the `.` at the end?
Also just want to confirm don't we need a changelog for #1240?

Reply:
there is a changelog for https://github.com/aws/amazon-ecs-agent/pull/1240

Reply:
will fix char count and `.` at the end. For changelog entry, I think adding it to release notes should suffice. @aaithal, @sharanyad?

Reply:
Sorry, I missed the last line. The changelog is already there.